Source/WebKit/Shared/RemoteLayerTree/RemoteLayerBackingStore.h

```diff
@@ -79,6 +79,8 @@ class RemoteLayerBackingStore : public CanMakeWeakPtr<RemoteLayerBackingStore> {
     RemoteLayerBackingStore(PlatformCALayerRemote*);
     virtual ~RemoteLayerBackingStore();
 
+    static std::unique_ptr<RemoteLayerBackingStore> createForLayer(PlatformCALayerRemote*);
```

Comment:
I think this can be just `create(PlatformCALayerRemote*)`. The argument type or name tells what this object is created for.